dll/win32/shlwapi/string.c

```diff
@@ -1068,7 +1068,13 @@ LPSTR WINAPI StrDupA(LPCSTR lpszStr)
 
   TRACE("(%s)\n",debugstr_a(lpszStr));
 
+#ifdef __REACTOS__
+  if (!lpszStr)
+    return NULL;
+  iLen = strlen(lpszStr) + 1;
```

Comment:
You could leave the `iLen` lines as they are, for minimal diff..

Reply:
Fixed.